Algebra/Module.swift

```diff
@@ -1,73 +0,0 @@
-//
-//  Module.swift
-//  Algebra
-//
-//  Created by Robert Widmann on 11/22/14.
-//  Copyright (c) 2014 TypeLift. All rights reserved.
-//
-
-/// If R is a Ring and M is an abelian group, a Left Module defines a binary operation R *<> M -> M.
-protocol LeftModule {
```

Comment:
I mean, we can always keep the protocols around but use an actual name for the module action.